cmd/juju/model/commit.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package model
+
+import (
+	"fmt"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+
+	"github.com/juju/juju/api/modelgeneration"
+	jujucmd "github.com/juju/juju/cmd"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/core/model"
+)
+
+const (
+	commitSummary = "Commits a branch to the model."
+	commitDoc     = `
+Committing a branch writes changes to charm configuration, charm URL and
```

Comment:
are the charmURL and resource pieces going to be implemented soon?  if not, suggested leaving those out for now to avoid confusion.

Reply:
Good point. I imagine the first incarnation released will only include deal with config. I will audit the other commands' docs too.